### PR TITLE
[infra/onert] Update cmake toolchain file

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit updates cmake toolchain file
- Remove tizen cross build toolchain file: not used
- Support clang build: Set USE_CLANG=1 environment variable
- Remove deprecated include: CMakeForceCompiler
- Remove using envoption()

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

File: infra/nnfw/cmake/buildtool/cross/toolchain_aarch64-linux.cmake [hunk 1]
```diff
@@ -1,20 +1,27 @@
...
 set(CMAKE_SYSTEM_PROCESSOR aarch64)
 
-set(CMAKE_C_COMPILER   aarch64-linux-gnu-gcc)
-set(CMAKE_CXX_COMPILER aarch64-linux-gnu-g++)
-
-# where is the target environment
-set(NNAS_PROJECT_SOURCE_DIR "${CMAKE_CURRENT_LIST_DIR}/../../../../..")
-set(ROOTFS_AARCH64 "${NNAS_PROJECT_SOURCE_DIR}/tools/cross/rootfs/aarch64")
-include("${NNAS_PROJECT_SOURCE_DIR}/infra/cmake/modules/OptionTools.cmake")
+if(DEFINED ENV{USE_CLANG} AND "$ENV{USE_CLANG}" STREQUAL "1")
```
Comment (seockho-kim) [inline infra/nnfw/cmake/buildtool/cross/toolchain_aarch64-linux.cmake:9]: just a question, 
why did you add clang support?

Comment (hseok-oh) [reply infra/nnfw/cmake/buildtool/cross/toolchain_aarch64-linux.cmake:9]: - Sometimes someone request clang build (ex. oversea member)
- More strict frontend check than gcc, and gcc follows these check on version up

**Review by seockho-kim — APPROVED** [review]
LGTM

**Review by glistening — APPROVED** [review]
LGTM
